readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
I really like that, make it obvious that you have to use the `css`  helper everywhere with helpful warnings is 👌